package.json

```diff
@@ -48,7 +48,10 @@
         "storybook:build": "yarn storybook:stories:build && build-storybook",
         "docs:analyze": "wca analyze 'packages/*/src/index.ts' --format json --outFile documentation/custom-elements.json",
         "docs:build": "yarn build && yarn docs:ci",
+        "docs:deploy": "yarn build && yarn docs:analyze && gulp docsDeploy && yarn storybook:build",
```

Comment:
should there be a README change in this PR about this? Any access people need to do these deploys?

Reply:
Oh this may be a command naming question. This is "build for deploy to the root director of a server" not "build and deploy". Thoughts on better naming?

Reply:
As this specifically would be used for deploying a staging version of the docs for reference in a PR, maybe `docs:build:staging`?

Reply:
`site:build`? Any difference between a staging build & a build from master outside of where it's deployed?

Reply:
The values in the `<base>` tag.

For the "production" site it uses `<base href="https://opensource.adobe.com/spectrum-web-components/">`, and for the "staging" site it uses `<base href="/">` which allows it to be hosted from the root dir rather than from `/spectrum-web-components`, etc.